configs/login-identity-providers-ldap.xml

```diff
@@ -65,7 +65,7 @@
     <provider>
         <identifier>ldap-provider</identifier>
         <class>org.apache.nifi.ldap.LdapProvider</class>
-        <property name="Authentication Strategy">SIMPLE</property>
+        <property name="Authentication Strategy">{{.Values.auth.ldap.authStrategy}}</property>
```

Comment:
I see the `master` branch already has the code you want (not hardcoded with SIMPLE. Not sure why the PR still shows SIMPE on the left pane??

Reply:
Hmm, I don't recall changing this file.
For people who do GitOps we need a existingSecret to be referred to.  Not using values. So the secrets aren't stored in git.